extra-cpp-bindings/Cargo.toml

```diff
@@ -11,7 +11,16 @@ crate-type = ["staticlib", "rlib", "cdylib"]
 [dependencies]
 anyhow = "1"
 cxx = "1"
-ethers = { version = "2", features = ["rustls"] }
+ethers = { version = "=2.0.0", features = ["rustls"] }
```

Comment:
why not use 2 instead?

Reply:
many compiling issues, continue checking